docs(content-releases): add frontend docs (#19391)
strapi/strapi: docs/docs/docs/01-core/content-manager/03-content-releases.mdx

---
title: Content Releases
description: Guide for content releases in the content-manager.
tags:
  - content-manager
  - content-releases
---

## Summary

Content releases is an enterprise feature so you need a valid license.
Additionally, your user role must be granted specific Permissions, such as `plugin::content-releases.read` and `plugin::content-releases.create-action`. You can modify these permissions in the Settings section. The capability to add entries to a release within the content manager is available through the content-manager edit view.

### Edit view

> If the feature is enabled with the correct license, and the user has the necessary permissions enabled, a 'Releases' section will appear on the right side of the Edit View. The 'Releases' section has an 'Add to release'. Clicking this button opens a modal allowing you to assign the entry to a specific release and specify the desired action (Publish/Unpublish).

Reviewer: ```suggestion
If the feature is enabled with the correct license, and the user has the necessary permissions enabled, a 'Releases' section will appear on the right side of the Edit View. The 'Releases' section has an 'Add to release' button. Clicking this button opens a modal allowing you to assign the entry to a specific release and specify the desired action (Publish/Unpublish).
```

> If the feature is enabled with the correct license, and the user role has the necessary permissions enabled, a 'Releases' sidebar will appear next to the edit view. Within this sidebar, you'll find an 'Add to release' button that, when clicked, opens a modal allowing you to assign the entry to a specific release and specify the desired action (Publish/Unpublish).

Reviewer: ```suggestion
If the feature is enabled with the correct license, and the user has the necessary permissions enabled, a 'Releases' section will appear on the right side of the Edit View. The 'Releases' section has an 'Add to release'. Clicking this button opens a modal allowing you to assign the entry to a specific release and specify the desired action (Publish/Unpublish).
```

In the event that no releases are present, users need to create a release first, which can be done on the [Content Releases](../content-releases/00-intro.md) page.

> If the user role also includes the `plugin::content-releases.delete-action` permission, the user can access additional options by clicking the 'More' button (represented by three dots) in the releases sidebar. This will reveal the 'Remove from release' button, providing the ability to detach the entry from the selected release.

Reviewer: ```suggestion
If the user's permissions also includes the `plugin::content-releases.delete-action`, the user can access additional options by clicking the 'More' button (represented by three dots) in the releases sidebar. This will reveal the 'Remove from release' button, providing the ability to detach the entry from the selected release.
```

Reviewer: I think it would be better if you always referred to the three dots button the same way in all files

'More' button (represented by three dots) vs the "three dots" button
